probe-rs/src/probe/stlink/constants.rs

```diff
@@ -89,6 +92,7 @@ pub enum Status {
     SwvNotAvailable = 0x20,
     JtagFreqNotSupported = 0x41,
     JtagUnknownCmd = 0x42,
+    Unknown = 0xFF,
```

Comment:
I think it's better to use `Other(u8)` here, but this will complicate status decoding.